pkg/scaling/utils/metrics.go

```diff
@@ -0,0 +1,39 @@
+package utils
```

Comment:
Could you please rename this package to `scaledjob`

Reply:
Done.